ci/forge/Dockerfile.forge

```diff
@@ -30,7 +31,8 @@ curl -fsSL https://deb.nodesource.com/setup_18.x | bash -
 
 apt-get update
 apt-get install -y \
-  awscli docker-ce-cli nodejs build-essential python-is-python3 python3-pip
+  awscli docker-ce-cli nodejs build-essential python-is-python3 \
+  python3-pip openjdk-8-jre wget
```

Comment:
do we need to install shellcheck somewhere?

how do we know that we are not missing any checks due to missing tools/dependencies?

Reply:
shellcheck is a part of the python/requirements/lint-requirements.txt file

Reply:
maybe consider change the script so that on ci it will fail when the required commands are missing.

Reply:
maybe as a TODO.

Reply:
TODO added to hard fail on the missing of lint tools